slirp4netns.c

```diff
@@ -279,6 +284,152 @@ Slirp *create_slirp(void *opaque, struct slirp4netns_config *s4nn)
     return slirp;
 }
 
+static int add_mount(const char *from, const char *to)
+{
+    int ret;
+
+    ret = mount(from, to, "", MS_BIND|MS_REC|MS_SLAVE|MS_NOSUID|MS_NODEV|MS_NOEXEC, NULL);
+    if (ret < 0) {
+        fprintf(stderr, "cannot bind mount %s to %s\n", from, to);
+        return ret;
+    }
+    ret = mount(from, to, "",  MS_REMOUNT|MS_REC|MS_BIND|MS_RDONLY|MS_NOSUID|MS_NODEV|MS_NOEXEC, NULL);
```

Comment:
`MS_REC|MS_RDONLY` isn't really read-only

Reply:
`MS_REC` is ignored in this case.  I've dropped it as it is not needed, thanks for noticing it